pkg/shares/shares_test.go

```diff
@@ -62,6 +64,32 @@ func TestMerge(t *testing.T) {
 	}
 }
 
+// TestPadFirstIndexedBlob ensures that we are adding padding to the first share
+// instead of calculating the value.
```

Comment:
[non-blocking] sorry I don't understand what "calculating the value" means in this context. Do you mind elaborating? It doesn't strictly have to go into the comment b/c I don't want to block this PR's merge.

Reply:
calculate using the non-interactive defaults, previously this is what it was doing. The estimation process was accurate enough that this was the case the vast majority of the time, but not for all cases.